packages/lwc/scripts/utils/rollup.js

```diff
@@ -38,6 +38,7 @@ function rollupConfig(config) {
                         preventAssignment: true,
                     }),
                 rollupFeaturesPlugin(prod),
+                // TODO [#2422]: remove rollup-plugin-typescript; we're just using it to transpile to ES5
```

Comment:
This is the one place we're still using `rollup-plugin-typescript`. I could not figure out how to get `@rollup/plugin-typescript` to emit exactly the same `dist` files as `rollup-plugin-typescript`. It kept using modern ES instead of ES5.

We should probably just be using Babel here instead, since all we want to do is transpile to ES5, but that feels like a risky change so I punted on it. In any case, to unblock things like `const enum` (W-9611979), we don't care if the `lwc` package is still using `rollup-plugin-typescript`.